bin/tasks.js

```diff
@@ -4,12 +4,12 @@ const path = require('path');
 const cwd = process.cwd();
 const target = path.resolve(cwd, 'public');
 const source = path.resolve(__dirname, '../src');
-const sass = require('witch')('node-sass');
+const npmsass = require('npm-sass');
```

Comment:
we should either use npmsass programmatically or just use the name of the binary